### PR TITLE
Skeleton improvements

### PR DESCRIPTION
**Pull request by SAnatoliiS**

### CODE REVIEW

File: kit/layout/src/Skeleton/types.ts [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+     *    You need to use the `Skeleton` as a separate component.
+     *    You will have to set a style that defines the dimensions of the `Skeleton`.
+     *    It is no necessary to pass `children`.
+     *
+     *    ```ts
+     *        <View>
+     *            {loading ? <Skeleton show style={contentSize} /> : <YourContent /> }
+     *        </View>
+     *    ```
+     */
```
Comment (sertony) [inline kit/layout/src/Skeleton/types.ts:42]: Great description!

File: kit/layout/src/Skeleton/Skeleton.tsx [hunk 2]
```diff
@@ -245,6 +199,7 @@ export function UISkeleton({
 
 const styles = StyleSheet.create({
     container: {
+        overflow: 'hidden',
```
Comment (sertony) [inline kit/layout/src/Skeleton/Skeleton.tsx:202]: Why wasn't it required before?

**Review by sertony — APPROVED** [review]
Will you integrate this in Surf as well? AFAIR it requires some changes there as well due to different usage there (as it was relaying on default border radius in some places)

File: kit/layout/src/Skeleton/Skeleton.tsx (hunk 2 above)
Comment (savelichalex) [reply kit/layout/src/Skeleton/Skeleton.tsx:202]: We dropped it back and forth due to bad initial API design :D

**Review by savelichalex — APPROVED** [review]

**Comment by vetkamak** [comment]
> Will you integrate this in Surf as well? AFAIR it requires some changes there as well due to different usage there (as it was relaying on default border radius in some places)

@sertony I have a task for it :)

**Pull request merged**